apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-action.tsx

```diff
@@ -81,11 +84,19 @@ export const InAppAction = () => {
             </Button>
           </DropdownMenuTrigger>
         </div>
-        <DropdownMenuContent className="p-1" align="end">
+        <DropdownMenuContent
+          className="p-1"
+          align="end"
+          onBlur={(e) => {
+            e.preventDefault();
+            e.stopPropagation();
+          }}
```

Comment:
weird behaviour but it was raising `onBlur` when hovering over the menu items

Reply:
Please add it as a comment.